content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -141,12 +142,37 @@ spec:
   driver: hostpath.csi.k8s.io
   source:
     snapshotHandle: 7bdd0de3-aaeb-11e8-9aae-0242ac110002
+  sourceVolumeMode: Filesystem
   volumeSnapshotRef:
     name: new-snapshot-test
     namespace: default
 ```
 
 `snapshotHandle` is the unique identifier of the volume snapshot created on the storage backend. This field is required for the pre-provisioned snapshots. It specifies the CSI snapshot id on the storage system that this `VolumeSnapshotContent` represents.
+`sourceVolumeMode` is the mode of the volume from which this snapshot was created. 
```

Comment:
It's possible that a user may be reading this doc but is using an older version of CRDs and controllers.  So we need to add notes right next to this field to clarify how it can be used.